### DISCUSSION TITLE
[M-Z] (17 Giugno 2019) Esercizio 5

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#485] [Esami & Esoneri M-Z]
### Traccia

<img width="620" alt="image" src="https://github.com/user-attachments/assets/cf90ab97-b3a4-4e54-a71a-010177ccb3fd">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190617.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190617.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/422ef639-1620-434a-b400-661d53e4b0c5)
